calculator.py

```diff
@@ -0,0 +1,22 @@
+"""
+This script function to calculate numerical values that performs
+addition, substraction, division and multiplication
+This function accepts user input ie the 2 numerical values
+and return the result value
+"""
+ 
+# perform the addition function
+def addition(value_1,value_2):

```

Comment:
- Adding a type hint will create a clear parameter for the function (which also will help the developer during the development process)
- Example:
```python
def combine_name(name_1: str, name2:str) -> str:
  return name_1 + " " + name_2
```
- In the example above `str` after `name_1` is the paramter type hint and `-> str` is the type return.

Reply:
Thank you! Alright will amend my code accordingly.